galaxy_ng/app/access_control/statements/pulp_container.py

```diff
@@ -155,7 +155,7 @@
                 "condition": "has_namespace_or_obj_perms:container.view_containerpushrepository",
             },
             {
-                "action": ["tag", "untag", "remove_image"],
+                "action": ["tag", "untag", "remove_image", "sign", "remove_signatures"],
```

Comment:
can you sign container repositories too, or just container push?

Reply:
Yes, container repositories works too.